packages/material-ui/src/Select/Select.js

```diff
@@ -101,6 +101,10 @@ Select.propTypes = {
   classes: PropTypes.object.isRequired,
   /**
    * If `true`, the selected item is displayed even if its value is empty.
+   *
+   *
+   *
+   * If you want to show a meaningful display, the `renderValue` property must be used to check for the case of an empty selected value and provide a custom render value.
    * You can only use it when the `native` property is `false` (default).
```

Comment:
At some point, I think that we will need to replace *property* with *prop* globally in the codebase.

Reply:
Yeah I tried to do that but gave up after 30minutes in which I only covered a fraction. You have to manually go through all of them at which point it doesn't make much sense.

Reply:
I remember discussing it before and did think of it when suggesting the edit, and did a little search and saw the attrition

Reply:
I'm sorry about that. I'm the one to blame for writting property over prop everywhere 😁.